### PR TITLE
Auto fill server

### PR DESCRIPTION
**Pull request by cameronmwall**

### CODE REVIEW

File: frontend/src/routes/ClusterManagement/Clusters/ImportCluster/ImportCluster.tsx [hunk 1]
```diff
@@ -210,6 +210,12 @@ export function ImportClusterPageContent() {
                     <AcmSubmit
                         id="submit"
                         variant="primary"
+                        // isDisabled={
```
Comment (zkayyali812) [inline frontend/src/routes/ClusterManagement/Clusters/ImportCluster/ImportCluster.tsx:213]: Why Is this commented out? Should it be removed?

Comment (cameronmwall) [reply frontend/src/routes/ClusterManagement/Clusters/ImportCluster/ImportCluster.tsx]: These should not have been commented out. Was left in there from fixing unit tests. Has been corrected

**Review by swopebe — APPROVED** [review]
I may be missing it, but I don't see any labels or messages to review so I will approve.

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**